src/ui/components/SecondaryToolbox/ReduxDevTools.tsx

```diff
@@ -79,6 +86,41 @@ export const ReduxDevToolsPanel = () => {
   );
 };
 
+async function isInsideApplyMiddlwareFn(
```

Comment:
:hand: It'll be smoother if we do the `applyMiddleware` function lookup once, then have a separate function that does the "is frame in function declaration" comparison synchronously